code/game/atoms_movable.dm

```diff
@@ -285,8 +285,14 @@
 			var/area/A = get_area_master(destination)
 			A.Entered(src)
 
-		for(var/atom/movable/AM in loc)

```

Comment:
<Clusterfack_> also no modifying base level procs
<Clusterfack_> for things for specific items